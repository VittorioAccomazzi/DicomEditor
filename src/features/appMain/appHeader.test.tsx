
import React from 'react';
import { cleanup, render } from '@testing-library/react'
import AppHeader from './appHeader'

beforeEach(() => {
  // setup a DOM element as a render target
});

afterEach(()=>{
  cleanup()
  jest.clearAllMocks()
})

test('rendering',()=>{
    const {container} =  render(<AppHeader numDicomFiles={50}/>)
    expect(container.innerHTML).toMatchSnapshot()
})