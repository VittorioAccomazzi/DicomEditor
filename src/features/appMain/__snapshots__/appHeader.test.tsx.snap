// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `"<div class=\\"MuiPaper-root makeStyles-paper-1 MuiPaper-elevation3 MuiPaper-rounded\\"><div class=\\"MuiBox-root MuiBox-root-14\\"><p class=\\"MuiTypography-root MuiTypography-body1 MuiTypography-colorPrimary\\">Number of Image Files: </p><p class=\\"MuiTypography-root makeStyles-boldtext-2 makeStyles-rightMargin-5 MuiTypography-body1\\">50</p><span class=\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-15 MuiCheckbox-root MuiCheckbox-colorPrimary makeStyles-noPadding-4 makeStyles-leftMargin-6 PrivateSwitchBase-checked-16 Mui-checked MuiIconButton-colorPrimary\\" aria-disabled=\\"false\\"><span class=\\"MuiIconButton-label\\"><input class=\\"PrivateSwitchBase-input-18\\" type=\\"checkbox\\" data-indeterminate=\\"false\\" value=\\"\\" checked=\\"\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></span><p class=\\"MuiTypography-root makeStyles-rightMargin-5 MuiTypography-body1 MuiTypography-colorPrimary\\"> replace current UIDs</p><button class=\\"MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-topBottomMarging-7 makeStyles-leftMargin-6 MuiButton-containedPrimary\\" tabindex=\\"0\\" type=\\"button\\"><span class=\\"MuiButton-label\\">  Download Images </span><span class=\\"MuiTouchRipple-root\\"></span></button></div></div>"`;
